src/pages/feature/heehyun/PostAdd/PostAdd.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import './PostAdd.scss';
+import { useNavigate } from 'react-router-dom';
+
+const PostAdd = () => {
+  const token = localStorage.getItem('TOKEN');
+
+  const navigate = useNavigate();
+
+  const [contents, setContents] = useState('');
+
+  const setChangeContents = (event) => {
+    setContents(event.target.value);
+  };
+
+  const goToBack = () => {
+    navigate('/main');
+  };
+
+  const letPost = () => {
+    fetch('http://10.58.52.224:8000/posts/insertThread', {
```

Comment:
API주소가 필요한 곳이 많아지고 프로젝트의 특성상 여러명의 백엔드 서버에 연결할 일이 생기다보면 기능마다 다른API주소가 사용되버려서 뜻하지않은 오류를 만날 가능성이 높아집니다. 이럴때는 API주소 부분만 따로 전역 변수로 생성해 관리하는편이 좋습니다
```suggestion
export const BASE_API  = "http://10.58.52.224:8000"
.
.
.
.
import {BASE_API} from '../../config.js'
fetch(`${BASE_API}/posts/insertThread`)
```